eng/pipelines/templates/stages/1es-redirect.yml

```diff
@@ -66,8 +66,7 @@ extends:
         justificationForDisabling: 'ESLint injected task has failures because it uses an old version of mkdirp. We should not fail for tools not controlled by the repo. See: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3499746'
       codeql:
         compiled:
-          enabled: false
-          justificationForDisabling: CodeQL times our pipelines out by running for 2+ hours before being force canceled.
+          enabled: true
```

Comment:
Given that codeql is only checking the source code I would like to see if we can disable it from all our normal pipelines and only enable it on our nightly aggregate-pipeline runs. That way we are still getting coverage but we don't have to worry about the times it wants to run for hours in our normal pipelines.